src/cart/swim/swim.c

```diff
@@ -48,6 +50,15 @@ swim_ping_timeout_default(void)
 	return val;
 }
 
+static inline int
+swim_subgroup_size_default(void)
+{
+	unsigned int val = SWIM_SUBGROUP_SIZE;
+
+	d_getenv_uint("SWIM_SUBGROUP_SIZE", &val);
```

Comment:
please add new env to CRT_ENV_LIST so that it gets dumped at the startup at info level. ideally we would change this (and other swim envs) to use crt_env_get() instead, but that would require include of crt_internal_types.h

Reply:
Sure, added to `CRT_ENV_LIST`. Sounds like you don't want swim.c to include crt_internal_types.h, so I'm not changing to use `crt_env_get`.